src/showcase/src/pages/components/PickAnchor.astro

```diff
@@ -0,0 +1,63 @@
+---
+import "$src/styles/main.css";
```

Comment:
You should import the css in a component like a header that way your do not have to add an import in each component

Reply:
Can you clarify what you mean? Then I would have to import that new component on every page here, right?

Reply:
Yes but I assumed that there are other things that are common and that can fits either a `<Layout />` or `<Head />` component.

Reply:
We do this [elsewhere](https://github.com/dfinity/internet-identity/blob/f3abe415ed8208ee8c0cad9ff923cafade48a89d/src/showcase/src/layouts/Layout.astro) but here it's a bit more complicated because the "components" (like `PickAnchor.astro`) are used as both components (imported in `components/index.astro`) and full blown pages. Having e.g. `PickAnchor.astro` wrapped in a Layout would mean the index would also include the layout, possibly several times

Reply:
Ok I had a long discussion with myself and I'll actually move the components out of `pages/` and will import them with the existing `Layout`; additional to the CSS they're other things (like `<meta name="viewport" content="width=device-width, initial-scale=1.0" />`) that we'd have to repeat everywhere